### ISSUE TITLE
Read Json Object from HTTPEndpoint Incoming request from another API

### ISSUE THREAD

**Issue by Rathna314** [elsa-workflows/elsa-core#4977]
How do I read the json sent from another API to ELsa Workflow V2 Dashboard HTTPEndPoint. Need some sample in Json.

**Comment by Rathna314** [comment]
Can someone respond please. Is "type": "RunJavaScript", available in Version 2.13 Dashboard. If not which activity should I use. I am unable to see RunJavaScript in 2.13 version.

**Comment by sfmskywalker** [comment]
Yes, but it's not added to the system by default. To make it available, you need to call `AddJavaScriptActivities` on the elsa options builder.

**Comment by sfmskywalker** [comment]
[This guide](https://v2.elsaworkflows.io/docs/guides/document-approval#http-endpoint) covers a similar scenario where it demonstrates reading a payload sent to an HTPT Endpoint activity.
